.github/settings.yml

```diff
@@ -0,0 +1,22 @@
+# The settings are inherited from common repository repo-settings (same file name and location)
+# You can append settings that are deeply merged with the inherited settings.
+#
+# When you have installed the GitHub App "repo-settings" in this repository,
+# any change of this settings.yml file is detected by the GitHub App and
+# the settings of this repository are updated immediately.
+#
+_extends: repo-settings:.github/common-settings-v2.yml
+
+# repo-specific settings
+#
+repository:
+  # See https://terraform-ibm-modules.github.io/documentation/#/implementation-guidelines?id=module-names-and-descriptions
+  name: "terraform-ibm-vpc"
+
+  # The description is displayed under the repository name on the
+  # organization page and in the 'About' section of the repository.
+
+  description: "Provisions and configures IBM Cloud"
```

Comment:
this makes no sense - did you mean to say "Provisions and configures IBM Cloud VPC resources"?

Reply:
😅 actually yes, added VPC resources